cpu/mips_pic32mx/include/cpu_conf.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+#ifndef _CPU_CONF_H_
+#define _CPU_CONF_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef THREAD_EXTRA_STACKSIZE_PRINTF
+#define THREAD_EXTRA_STACKSIZE_PRINTF   (2048)
+#endif
+#ifndef THREAD_STACKSIZE_DEFAULT
+#define THREAD_STACKSIZE_DEFAULT        (4096)
```

Comment:
Well code built at -O0 is very stupid and always uses the stack, so stack usage is always higher in debug builds than release.

What symbol(s) do you use to differentiate between a debug vs release build in RIOT ?